modules/peers.js

```diff
@@ -156,10 +185,25 @@ __private.insertSeeds = function (cb) {
 	library.logger.trace('Peers->insertSeeds');
 	async.each(library.config.peers.list, function (peer, eachCb) {
 		peer = library.logic.peers.create(peer);
-		library.logger.trace('Processing seed peer: ' + peer.string);
-		self.ping(peer, function (err) {
-			++updated;
-			return setImmediate(eachCb);
+		library.logger.debug('Processing seed peer: ' + peer.string);
+		if (_.isEmpty(self.acceptable([peer]))) {
+			return setImmediate(eachCb, 'Peer not accepted: ' + peer.string);
+		}
+		peer.rpc.status(function (err, status) {
+			if (err) {
+				peer.applyHeaders({state: Peer.STATE.DISCONNECTED});
+				library.logger.trace('Ping peer failed: ' + peer.string, err);
+			} else {
+				peer.applyHeaders({
+					height: status.height,
+					broadhash: status.broadhash,
+					nonce: status.nonce,
+					state: Peer.STATE.CONNECTED //connected
```

Comment:
after discovery phase there could be the situation that disconnected peer was drawn that is online now - we want to mark it as connected then